bld/namelist_files/namelist_definition_mpassi.xml

```diff
@@ -3715,3 +3728,4 @@ Default: Defined in namelist_defaults.xml
 </entry>
```

Comment:
Is this `config_prescribed_ice_stream_fldfilename` entry being properly closed now? I think it should also have the `Valid values:` and `Default:` keys.

Reply:
Added.